src/Analyzers/CSharp/Analyzers/UseImplicitOrExplicitType/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -50,7 +57,7 @@ private void HandleVariableDeclaration(SyntaxNodeAnalysisContext context)
             var options = context.Options;
             var syntaxTree = context.Node.SyntaxTree;
             var cancellationToken = context.CancellationToken;
-            var optionSet = options.GetDocumentOptionSetAsync(syntaxTree, cancellationToken).GetAwaiter().GetResult();
+            var optionSet = options.GetOptions(syntaxTree, cancellationToken);
```

Comment:
love these changes.   can we do this *independently* of this PR?

Reply:
Yeah, @JoeRobich has a separate PR for these changes, which hopefully will get merged soon :)

Reply:
fantastic.  as many changes as we can make independently would help out a lot here.

Reply:
Awesome.  Let's def have that done first :)